R/dust.R

```diff
@@ -1,4 +1,4 @@
-## Generated by dust (version 0.9.15) - do not edit
+## Generated by dust (version 0.9.11) - do not edit
```

Comment:
Could you update your dust/odin.dust (to what's on drat) and recompile the odin code please

Reply:
done